jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -26,6 +26,7 @@
 #include "ecma-objects-arguments.h"
 #include "ecma-try-catch-macro.h"
 #include "jcontext.h"
+#include "jerryscript-core.h"
```

Comment:
I just realized this is very good for the project since we can do all external_pointer_t casts and let the compiler to do its checks. The patch is a bit bigger but well worth it.